inc/blocks/ab-test/register.php

```diff
@@ -113,6 +113,30 @@ function render_block( array $attributes, ?string $inner_content = '' ) : string
 		$winner = $results['winner'] ?? false;
 	}
 
+	if ( is_preview() || isset( $_GET['set_test'] ) ) {
+		wp_enqueue_style(
+			'altis-experiments-features-blocks-abtest',
+			plugins_url( 'inc/blocks/ab-test/edit.css', Analytics\ROOT_FILE ),
+			[],
+			null
+		);
+		wp_enqueue_script(
+			'ab-test-preview',
+			Utils\get_asset_url( 'blocks/ab-test-preview.js' ),
+			[],
```

Comment:
When using `wp.i18n` functions in a script you need to ensure that `wp-i18n` is in the the list of dependencies for the script when you enqueue it.